recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 1002
```

Comment:
So shouldn't this be 1001 then?

Reply:
If we want a new package for the legacy compilers as well, we need to bump `1` more. As it goes `1001 -> 1` and `1002 -> 2` for the legacy compilers. We would use `1001` if we only wanted a new package for the new compilers and not the legacy compilers. This will give us a package for both the legacy and the new compilers. HTH